public/_includes/page-layout/top_nav.html

```diff
@@ -0,0 +1,35 @@
+<nav id="top-navigation" class="navbar navbar-default">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#mobileNav">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/" title="CyberArk Logo"></a>
+    </div>
+    <div class="collapse navbar-collapse" id="mobileNav">
+      <ul class="nav navbar-nav">
+
+        {% for item_hash in site.data.navigation.main %}
+          {% assign key = item_hash[0] %}
+    			{% assign value = item_hash[1] %}
+    			{% include page-layout/top_nav_menu.html item=value section=key %}
+    		{% endfor %}
+
+        <li class="nav-item">
```

Comment:
Should we add a "Docs" link to docs.conjur.org to the right nav?  Or Even just conjur.org?

Reply:
I would like to have the nav logo point to conjur.org ... but haven't figured out how yet 😬